certbot/cli.py

```diff
@@ -850,6 +850,12 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
         None, "-t", "--text", dest="text_mode", action="store_true",
         help=argparse.SUPPRESS)
     helpful.add(
+        None, "--log-backups", type=nonnegative_int, default=1000,
```

Comment:
Naming is tough, I know, but a `-max-` might be prudent here, to make it a bit more clear that it's not how many copies of the log we're making.